File: Linguagem C/crescente.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int a,b;


    printf("Digite dois numeros: \n");
    scanf("%d", &a);
    scanf("%d", &b);
    if (a != b)
    {
        if (a > b)
        {
            printf("Decrescente!\n");
        }
        else
        {
            printf("Crescente!\n");
        }
    }

    while (a != b) {
        printf("Digite outros dois numeros: \n");
        scanf("%d", &a);
        scanf("%d", &b);
        if (a != b) {
            if (a > b) {
                printf("Decrescente!\n");
            }
            else {
                printf("Crescente!\n");
            }
        }
  }
   return 0;
}

File: Linguagem Java/Crescente.java
import java.util.Locale;
import java.util.Scanner;

public class Crescente {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite dois numeros: ");
		int x = sc.nextInt();
		int y = sc.nextInt();
		if (x != y) {
			while (x != y) {
				if (x < y) {
					 System.out.println("Crescente");
				}
				else {
					System.out.println("Decrescente");
				}
				System.out.println("Digite outros dois numeros: ");
				x = sc.nextInt();
				y = sc.nextInt();
			}
			
		}
		
		
	}

}
